### PR TITLE
add package: cairo-dock

### PR DESCRIPTION
**Pull request by Yisus7u7**
Screenshot_20211124-135842
Screenshot_20211124-154317
Screenshot_20211123-192233

even if this is only the core, without its beautiful accessories, it is fully functional

> Note: It is recommended to run it with the option "do not use openGL" if you want go with a good performance

### CODE REVIEW

**Comment by Yisus7u7** [comment]
@xeffyr since I make it not use ninja, it should use is "Makefile / make"

**Comment by Yisus7u7** [comment]
The compilation was successful, the installation failed, how do I make it use  default make?

File: packages/cairo-dock-core/build.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+_COMMIT=6c569e67a2a366e7634224a0133ede51755629cb
+TERMUX_PKG_SRCURL=https://github.com/Cairo-Dock/cairo-dock-core/archive/${_COMMIT}.zip
+TERMUX_PKG_SHA256=e59e99147ce9c901b46d4b56b88bd53aeda34292b86fd9fbf2c55d158153f2ec
+TERMUX_PKG_DEPENDS="atk, dbus-glib, glib, gtk3, pango, libcairo, libx11, xsltproc, libcurl, gdk-pixbuf, harfbuzz, mesa, glu, librsvg, libxcomposite, libwayland, libwayland-protocols, libxinerama, libxrandr, libxrender, libxtst, libxml2"
+TERMUX_PKG_BUILD_DEPENDS="valac, cmake"
+TERMUX_PKG_BUILD_IN_SRC=true
+termux_step_configure_(){
+	cd ${TERMUX_PKG_SRCDIR}
+	mkdir build && cd build
+	${TERMUX_PREFIX}/bin/cmake .. -DCMAKE_INSTALL_PREFIX=${TERMUX_PREFIX}
```
Comment (xeffyr) [inline packages/cairo-dock-core/build.sh:15]: Don't use tools in `$TERMUX_PREFIX`.

**Review by xeffyr — CHANGES_REQUESTED** [review]

**Comment by xeffyr** [comment]
`TERMUX_CMAKE_BUILD="Unix Makefiles"`.

Also forget about things that work for on-device builds. Official builder image doesn't use Termux and neither supports execution of cross-compiled binaries and Android in general.

If you still want to specific steps for on-device compilation, surround them with
```
if $TERMUX_ON_DEVICE_BUILD; then
# your commands there
fi
```

Don't reference build tools by absolute path, especially via `$TERMUX_PREFIX/bin/`. This should be handled by shell through `PATH` variable.

File: packages/cairo-dock-core/build.sh [hunk 2]
```diff
@@ -12,5 +12,5 @@ TERMUX_PKG_BUILD_IN_SRC=true
 termux_step_configure_(){
 	cd ${TERMUX_PKG_SRCDIR}
 	mkdir build && cd build
-	${TERMUX_PREFIX}/bin/cmake .. -DCMAKE_INSTALL_PREFIX=${TERMUX_PREFIX}
+	/usr/bin/cmake .. -DCMAKE_INSTALL_PREFIX=${TERMUX_PREFIX}
```
Comment (xeffyr) [inline packages/cairo-dock-core/build.sh:15]: You are not supposed to use cmake in such way...

**Review by xeffyr — CHANGES_REQUESTED** [review]

**Comment by Yisus7u7** [comment]
> `TERMUX_CMAKE_BUILD="Unix Makefiles"`.
> 
> Also forget about things that work for on-device builds. Official builder image doesn't use Termux and neither supports execution of cross-compiled binaries and Android in general.
> 
> If you still want to specific steps for on-device compilation, surround them with
> 
> ```
> if $TERMUX_ON_DEVICE_BUILD; then
> # your commands there
> fi
> ```
> 
> Don't reference build tools by absolute path, especially via `$TERMUX_PREFIX/bin/`. This should be handled by shell through `PATH` variable.

didn't work, did he use ninja, or did I do it wrong?

**Pull request merged**